src/shared/mediaTypesUtils/multipartAcceptHeaderFieldValue.js

```diff
@@ -0,0 +1,50 @@
+import assertMediaTypeIsValid from './assertMediaTypeIsValid.js';
+import { mediaTypes } from './mediaTypes.js';
+
+/**
+ * Builds an accept header field value for HTTP GET multipart request messages.
+ *
+ * Takes in input a media types array of type [{mediaType, transferSyntaxUID}, ... ]
+ * and finally composes a string for the accept header field value as in example below:
+ *
+ * "multipart/related; type="image/x-dicom-rle"; transfer-syntax=1.2.840.10008.1.2.5,
+ *  multipart/related; type="image/jpeg"; transfer-syntax=1.2.840.10008.1.2.4.50,
+ *  multipart/related; type="application/octet-stream"; transfer-syntax=*"
+ *
+ * NOTE: the xhr request will try to fetch with all the transfer-syntax syntaxes
+ * specified in the accept header field value in descending order.
+ * The first element ("image/x-dicom-rle" in this example) has the highest priority.
+ *
+ * @param {Array} mediaTypes Acceptable media types
+ *
+ * @returns {string} accept header field value
+ */
+
+export default function buildMultipartAcceptHeaderFieldValue(mediaTypes) {
+  if (!Array.isArray(mediaTypes)) {
+    throw new Error('Acceptable media types must be provided as an Array');
+  }
+
+  const fieldValueParts = [];
+
+  mediaTypes.forEach((item) => {
+    const { transferSyntaxUID, mediaType } = item;
+
+    assertMediaTypeIsValid(mediaType);
+
+    let fieldValue = `multipart/related; type="${mediaType}"`;
```

Comment:
this should not have `type="${mediaType}"` it should be `type=${mediaType}`. Double checking with @wayfarer3130

Reply:
Hello @sedghi,
Referring to [azure dicom docs](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/healthcare-apis/dicom/dicom-services-conformance-statement.md#retrieve-an-instance), type shoud be in double quotes. Otherwise, 406 response status will be returned.

Reply:
According to the DICOM standard it depends on the version, and according to the original RFC for multipart related, the quotes are NOT permitted, although every example in the document shows them.  Including them drops the speed by approximately a half because it requires a full extra CORS request, assuming the CORS is used.  Not including htem breaks Azure, so the DICOM committee has agreed to a statement that clients need to have an option to fallback from one version to the other.  For cornerstone, that is determined by a configuration parameter on the specific back end request, so that you can include them when configured to do so, and omit them otherwise.